docs(grid): revamp get current grid data kb article (#1720)
telerik/blazor-docs: knowledge-base/grid-get-filtered-data.md

---
title: Get Filtered and Sorted Data from Grid
description: How to get the current filtered and sorted data from the Telerik Blazor Grid. How to get the correct table row index when the Grid data state doesn't match the order in the original data source.
type: how-to
page_title: How to Get the Filtered and Sorted Data from the Grid
slug: grid-kb-get-filtered-data
position: 
tags: 
ticketid: 1629418, 1627421, 1626964, 1617119, 1561446, 1534078, 1489807
res_type: kb
---

## Environment

<table>
    <tbody>
        <tr>
            <td>Product</td>
            <td>Grid for Blazor</td>
        </tr>
    </tbody>
</table>

## Description

> This KB article answers the following questions. All of them revolve around the Grid table rows, which the component is currently displaying and which the user sees.

Author: ```suggestion
This KB article answers the following questions. All of them revolve around the Grid table rows, which the component is currently displaying to the user.
```
